app/main.py

```diff
@@ -38,3 +38,18 @@ async def index():
 def blocking():
     time.sleep(2)
     return {"ret": 42}
+
+# Get https://phinvads.cdc.gov/baseStu3/CodeSystem/ and return a whole code system result
+@app.get("/phinvads/CodeSystem")
+@cache(namespace="phinvads", expire=3600)
+def get_code_systems(code: str = None):
+    url = "https://phinvads.cdc.gov/baseStu3/CodeSystem"
```

Comment:
can you update to use `pv_base_url` (added in latest merge to main)?

Reply:
Yep, will update all of this (none was yet merged when I first drafted this).